controllers/vrg_kubeobjects.go

```diff
@@ -364,7 +370,24 @@ func (v *VRGInstance) kubeObjectsRecover(s3ProfileName string, objectStore Objec
 		kubeObjectsRequestNameLabelKey, recoverNamePrefix)
 }
 
+// return true = disabled
+func (v *VRGInstance) kubeObjectProtectionDisabled() bool {
+	const defaultState = false
+
+	_, ramenConfig, err := ConfigMapGet(v.ctx, v.reconciler.APIReader)
+	if err != nil {
+		return defaultState
+	}
+
+	if ramenConfig.KubeObjectProtection != nil && ramenConfig.KubeObjectProtection.Disabled != nil {
+		return *ramenConfig.KubeObjectProtection.Disabled
+	}
+
+	return defaultState
+}
+
 func (v *VRGInstance) kubeObjectProtectionDisabledOrKubeObjectsProtected() bool {
 	return v.instance.Spec.KubeObjectProtection == nil ||
-		v.instance.Status.KubeObjectProtection.CaptureToRecoverFrom != nil
+		v.instance.Status.KubeObjectProtection.CaptureToRecoverFrom != nil ||
+		v.kubeObjectProtectionDisabled()
```

Comment:
If somehow `v.instance.Status.KubeObjectProtection.CaptureToRecoverFrom`  were `!= nil`, then `v.kubeObjectProtectionDisabled()` would not be executed.  Not sure the impact, but I think the intent is to call it.  One way to address it is by moving up `v.kubeObjectProtectionDisabled() ||` to the beginning.